src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -807,12 +807,15 @@ protected override SyntaxDiagnosticInfo GetExpectedTokenError(SyntaxKind expecte
             }
         }
 
-        protected override SyntaxDiagnosticInfo GetExpectedTokenError(SyntaxKind expected, SyntaxKind actual)
+        protected override SyntaxDiagnosticInfo GetExpectedMissingNodeOrTokenError(
+            GreenNode missingNodeOrToken, SyntaxKind expected, SyntaxKind actual)
         {
+            Debug.Assert(missingNodeOrToken.IsMissing);
```

Comment:
Renamed this api to make it clear what purpose it is for (producing an error when a token/node is expected but missing).

Also, changed this api to get the missingNodeOrToken passed in.  This way it can validate the callers are all using it properly.